<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label2">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>기능2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="onoff2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedControls">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label1">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>기능1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="onoff1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>100</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedControls">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>127</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="Off">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="On">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>On</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
